x-pack/legacy/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -8,49 +8,41 @@ import _ from 'lodash';
 import React, { useState, useEffect, useCallback, useRef } from 'react';
 import { I18nProvider } from '@kbn/i18n/react';
 import { i18n } from '@kbn/i18n';
-import { EuiLink, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import { Storage } from 'ui/storage';
 import { CoreStart, SavedObjectsClientContract } from 'src/core/public';
+import {
+  DataSetup,
+  IndexPattern as IndexPatternInstance,
+  SavedQuery,
+} from 'src/legacy/core_plugins/data/public';
+import { Filter } from '@kbn/es-query';
+import { TopNavMenu } from '../../../../../../src/legacy/core_plugins/kibana_react/public';
 import { Query } from '../../../../../../src/legacy/core_plugins/data/public/query';
```

Comment:
You can do this import from data/public (without query in the end, which won't work in NP)